<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabs">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>759</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionAutosave"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
    <addaction name="actionClose_all"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_file"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuFont">
     <property name="title">
      <string>Font</string>
     </property>
     <addaction name="actionBigger"/>
     <addaction name="actionSmaller"/>
     <addaction name="separator"/>
     <addaction name="actionFont_family"/>
    </widget>
    <widget class="QMenu" name="menuHeading">
     <property name="title">
      <string>Heading</string>
     </property>
     <addaction name="actionStandard"/>
     <addaction name="actionHeading_1"/>
     <addaction name="actionHeading_2"/>
     <addaction name="actionHeading_3"/>
     <addaction name="actionHeading_4"/>
     <addaction name="actionHeading_5"/>
     <addaction name="actionHeading_6"/>
    </widget>
    <widget class="QMenu" name="menuList">
     <property name="title">
      <string>List</string>
     </property>
     <widget class="QMenu" name="menuBullet_list">
      <property name="title">
       <string>Bullet list</string>
      </property>
      <addaction name="actionCircle"/>
      <addaction name="actionSquare"/>
      <addaction name="actionDisc"/>
     </widget>
     <widget class="QMenu" name="menuOrdered_list">
      <property name="title">
       <string>Ordered list</string>
      </property>
      <addaction name="actionAlpha_lower"/>
      <addaction name="actionAlpha_upper"/>
      <addaction name="actionRoman_lower"/>
      <addaction name="actionRoman_upper"/>
      <addaction name="actionStandard_numeric"/>
     </widget>
     <addaction name="menuBullet_list"/>
     <addaction name="menuOrdered_list"/>
     <addaction name="actionCheckbox"/>
     <addaction name="actionCheckbox_checked"/>
    </widget>
    <widget class="QMenu" name="menuText_alignment">
     <property name="title">
      <string>Text alignment</string>
     </property>
     <addaction name="actionLeft"/>
     <addaction name="actionRight"/>
     <addaction name="actionCenter"/>
     <addaction name="actionJustify"/>
    </widget>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="actionStrikeout"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="menuHeading"/>
    <addaction name="menuFont"/>
    <addaction name="menuList"/>
    <addaction name="separator"/>
    <addaction name="actionColor"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="menuText_alignment"/>
   </widget>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionStay_topmost"/>
    <addaction name="separator"/>
    <addaction name="actionRemeber_opened_files"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
    <addaction name="actionForce_Quit"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuProgram"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/exit_48px.png</normaloff>:/icons/icons/exit_48px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/bold.png</normaloff>:/icons/icons/bold.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/italic_48px.png</normaloff>:/icons/icons/italic_48px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/underline_48px.png</normaloff>:/icons/icons/underline_48px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Underline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/new_copy_48px.png</normaloff>:/icons/icons/new_copy_48px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/close_pane_48px.png</normaloff>:/icons/icons/close_pane_48px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionStrikeout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/strikethrough_48px.png</normaloff>:/icons/icons/strikethrough_48px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Strikeout</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionBigger">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/increase_font_48px.png</normaloff>:/icons/icons/increase_font_48px.png</iconset>
   </property>
   <property name="text">
    <string>Bigger</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+=</string>
   </property>
  </action>
  <action name="actionSmaller">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/decrease_font_48px.png</normaloff>:/icons/icons/decrease_font_48px.png</iconset>
   </property>
   <property name="text">
    <string>Smaller</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+-</string>
   </property>
  </action>
  <action name="actionFont_family">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/choose_font.png</normaloff>:/icons/icons/choose_font.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Font family</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+F</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/color_mode_48px.png</normaloff>:/icons/icons/color_mode_48px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="actionStay_topmost">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stay on top</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/folder_48px.png</normaloff>:/icons/icons/folder_48px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/close_all_tabs_48px.png</normaloff>:/icons/icons/close_all_tabs_48px.png</iconset>
   </property>
   <property name="text">
    <string>Close all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/save_48px.png</normaloff>:/icons/icons/save_48px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/save_as_48px.png</normaloff>:/icons/icons/save_as_48px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionDelete_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/delete_bin_24px.png</normaloff>:/icons/icons/delete_bin_24px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Del</string>
   </property>
  </action>
  <action name="actionForce_Quit">
   <property name="text">
    <string>Force &amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="actionAutosave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/save_close_48px.png</normaloff>:/icons/icons/save_close_48px.png</iconset>
   </property>
   <property name="text">
    <string>Autosave</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRemeber_opened_files">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/idea_48px.png</normaloff>:/icons/icons/idea_48px.png</iconset>
   </property>
   <property name="text">
    <string>Remember opened files</string>
   </property>
  </action>
  <action name="actionStandard">
   <property name="text">
    <string>Standard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+0</string>
   </property>
  </action>
  <action name="actionHeading_1">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/header_1_48px.png</normaloff>:/icons/icons/header_1_48px.png</iconset>
   </property>
   <property name="text">
    <string>Heading 1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+1</string>
   </property>
  </action>
  <action name="actionHeading_2">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/header_2_48px.png</normaloff>:/icons/icons/header_2_48px.png</iconset>
   </property>
   <property name="text">
    <string>Heading 2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+2</string>
   </property>
  </action>
  <action name="actionHeading_3">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/header_3_48px.png</normaloff>:/icons/icons/header_3_48px.png</iconset>
   </property>
   <property name="text">
    <string>Heading 3</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+3</string>
   </property>
  </action>
  <action name="actionHeading_4">
   <property name="text">
    <string>Heading 4</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+4</string>
   </property>
  </action>
  <action name="actionHeading_5">
   <property name="text">
    <string>Heading 5</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+5</string>
   </property>
  </action>
  <action name="actionHeading_6">
   <property name="text">
    <string>Heading 6</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+6</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/list.png</normaloff>:/icons/icons/list.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionSquare">
   <property name="text">
    <string>Square</string>
   </property>
  </action>
  <action name="actionDisc">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/list_48px.png</normaloff>:/icons/icons/list_48px.png</iconset>
   </property>
   <property name="text">
    <string>Disc</string>
   </property>
  </action>
  <action name="actionAlpha_lower">
   <property name="text">
    <string>Alpha lower</string>
   </property>
  </action>
  <action name="actionAlpha_upper">
   <property name="text">
    <string>Alpha upper</string>
   </property>
  </action>
  <action name="actionRoman_lower">
   <property name="text">
    <string>Roman lower</string>
   </property>
  </action>
  <action name="actionRoman_upper">
   <property name="text">
    <string>Roman upper</string>
   </property>
  </action>
  <action name="actionStandard_numeric">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/numbered_list_48px.png</normaloff>:/icons/icons/numbered_list_48px.png</iconset>
   </property>
   <property name="text">
    <string>Standard (numeric)</string>
   </property>
  </action>
  <action name="actionCheckbox">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/checklist.png</normaloff>:/icons/icons/checklist.png</iconset>
   </property>
   <property name="text">
    <string>Checkbox</string>
   </property>
  </action>
  <action name="actionCheckbox_checked">
   <property name="text">
    <string>Checkbox (checked)</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/align_left_24px.png</normaloff>:/icons/icons/align_left_24px.png</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/align_right_24px.png</normaloff>:/icons/icons/align_right_24px.png</iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/align_center_24px.png</normaloff>:/icons/icons/align_center_24px.png</iconset>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionJustify">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/align_justify_24px.png</normaloff>:/icons/icons/align_justify_24px.png</iconset>
   </property>
   <property name="text">
    <string>Justify</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+J</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
